### DISCUSSION TITLE
Convert pythonista flixy into macOS or windows app 🥳!!

### DISCUSSION THREAD

**Discussion by SKbarbon** [SKbarbon/flixy#5] [Announcements]
For the first time, you can now convert your pythonista `flixy` app into macOS and windows app.
And,,, guess what ‽!!!!!!!!!!
You can now use flixy on mac and windows as a normal library !!!!!!!!.

Just go to your mac or windows, and then type:
```cmd
pip install flixy2app
```

Then boom 🤯!! you have it !!

To convert your pythonista scripts into an macOS or windows executable file, just type:

```cmd
python3 -m flixy2app.generate
```

Follow the instructions, and you have your own app !!!!

Warning ⚠️!!
This is BETA !! So it will be so broken at the first releases. Please to support this project, create an issues if you find one!!.

There is a big limitations, too:
- There is no WebView support for now.
- You cannot use pythonista platform-specific libraries like `ui`.

### `flixy` is a pythonista library that kinda compiled on macOS and windows. So all features will support pythonista first. Then it will come to desktop.

Good luck, me :)

**Comment by nervosajj80** [comment]
Dude, you rock! I am super excited to see the progress that the Pythonista community has made in only the past weeks. It's amazing. Ole is back! My man here is working on making Pythonista portable! The future is looking brighter all the time! Lol😎
